0008-atomic-registry.md

```diff
@@ -0,0 +1,144 @@
+- Feature Name: Katello and Atomic Registry Integration
+- Start Date: 2016-08-11
+- RFC PR:
+- Issue:
+
+Table of Contents
+ * [Summary](#summary)
+ * [Benefits](#benefits)
+ * [Example](#example)
+ * [User Authentication and Role Based Access Control](#userauthentication)
+ * [Installation](#installation)
+ * [Capsules](#capsules)
+
+# Summary
+[summary]: #summary
+
+To facilitate a more natural feel for working with containers and [Katello](https://katello.org), [Atomic Registry](http://www.projectatomic.io/registry/) will become an optional integration point. This true registry will provide the full set of features required by users, while allowing Katello to continue to act as a container repository. Integration between the two will allow images in the Katello content management lifecycle to be directly linked to images in Atomic Registry projects, and vice versa.
+
+# Benefits
+[benefits]: #benefits
+
+The docker registry API provided by Katello does not support 'docker push' commands so it is effectively a read-only source of images. With the inclusion of Atomic Registry, the full set of docker commands will be available.
+
+When Katello imports a docker image into a product repository, it is given a name that includes the organization and the product. In addition, when this product repository is added to a content view and promoted, the image name will include this information as well. While this is necessary to uniquely identify these images within Katello it can be confusing to users expecting to access images by the original name. With Atomic Registry these images can be provided as their original names, if desired.
+
+
+# Example
```

Comment:
The examples are quite mechanical feeling, i.e. they explain to me how the data exchanges work but they don't give me a feel for the user workflows that this integration will solve.